www-apps/rutorrent/rutorrent-4.0.1.ebuild

```diff
@@ -0,0 +1,71 @@
+# Copyright 1999-2023 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=8
+
+inherit webapp optfeature
+
+DESCRIPTION="ruTorrent is a front-end for the popular Bittorrent client rTorrent"
+HOMEPAGE="https://github.com/Novik/ruTorrent"
+if [[ ${PV} == 9999 ]]; then
+	inherit git-r3
+	EGIT_REPO_URI="https://github.com/Novik/ruTorrent.git"
+else
+	MY_PV="${PV}-hotfix"
+	SRC_URI="https://github.com/Novik/ruTorrent/archive/v${MY_PV}.tar.gz -> ${P}.tar.gz"
+	KEYWORDS="~alpha ~amd64 ~ppc ~x86"
+	S="${WORKDIR}/ruTorrent-${MY_PV}"
+fi
+
+LICENSE="GPL-2+ MIT"
+
+RDEPEND="
+	dev-lang/php[xml,gd]
+	virtual/httpd-php
+"
+
+need_httpd_cgi
+
+pkg_setup() {
+	webapp_pkg_setup
+}
+
+src_install() {
+	webapp_src_preinst
+
+	rm -r .github || die
+	find . \( -name .gitignore -o -name .gitmodules \) -type f -exec rm '{}' + || die
+	if [[ ${PV} == 9999 ]]; then
+		rm -r .git .gitattributes || die
+	fi
+
+	insinto "${MY_HTDOCSDIR}"
+	doins -r .
+
+	chmod +x "${ED}${MY_HTDOCSDIR}"/plugins/*/*.sh \
+		"$ED${MY_HTDOCSDIR}"/php/test.sh || die "chmod failed"
```

Comment:
Please add a comment here that fperms does not work with globbing
(and also `${ED}` for the second `$ED`, to be consistent)